recipes/eigen/all/conanfile.py

```diff
@@ -63,19 +69,12 @@ def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "Eigen3")
         self.cpp_info.set_property("cmake_target_name", "Eigen3::Eigen")
         self.cpp_info.set_property("pkg_config_name", "eigen3")
```

Comment:
while we're here, I think we may need the prefixes `Eigen3` and `EIGEN3` for compat? I think we had an issue somewhere?

Reply:
Leaving as-is to avoid users from relying on legacy features